mungegithub/mungers/submit-queue.go

```diff
@@ -161,10 +162,12 @@ type SubmitQueue struct {
 
 	Committers             string
 	RequiredStatusContexts []string
-	doNotMergeMilestones   []string
+	DoNotMergeMilestones   []string
```

Comment:
General comment-- I would actually go further, and move all public settings to the top of their structs, with a comment in front of them that says "// flag values" or something else that makes it clear they are used as input for the munger and it's actually OK to set them (before first use, anyway) or read them afterwards.

And document the convention somewhere, maybe by the Munger interface.